common/contract.ts

```diff
@@ -66,6 +67,7 @@ export type Contract<T extends AnyContractType = AnyContractType> = {
   flaggedByUsernames?: string[]
   openCommentBounties?: number
   unlistedById?: string
+  itemType: 'contract'
```

Comment:
If you're adding this field as a non-optional type, you need to backfill existing contracts to have it by writing a script.

Reply:
I actually don't think this field is necessary. The only place it's used is `likeItem`.

You could simply pass in the itemType to that function. There's no need to store it in the document, it's redundant because we know which collection the document is in.

Another idea is to have functions `likeContract` and `likePost` which call `likeItem` with the appropriate itemType.

Reply:
Yea I had the script to backfill it setup, was gonna run it before shipping. 

But anyway, yep that makes sense, just updated to pass the itemType along.